src/mmg3d/mmg3d1.c

```diff
@@ -846,7 +867,10 @@ MMG5_int MMG5_movtet(MMG5_pMesh mesh,MMG5_pSol met, MMG3D_pPROctree PROctree,
               }
               ier = MMG5_movbdyregpt(mesh,met,PROctree,listv,ilistv,
                                      lists,ilists,improveSurf,improveVolSurf);
-              if (ier < 0 ) return -1;
+              if (ier < 0 ){
+                MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```